Oswaldo Vazquez Aparicio/30 Agosto/hackerrank/hashtablesransomnote/Result.java

```diff
@@ -25,8 +26,8 @@ public static void checkMagazine(List<String> magazine, List<String> note) {
 
     }
 
-    private static HashMap<String, Integer> createWordsFrequencyTable(List<String> words){
-        HashMap<String, Integer> frequencyTable = new HashMap<>();
+    private static Map<String, Integer> createWordsFrequencyTable(List<String> words){
+        Map<String, Integer> frequencyTable = new HashMap<>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...